apps/designer/app/designer/features/style-panel/shared/style-info.ts

```diff
@@ -1,9 +1,43 @@
-import type { Style, StyleProperty, StyleValue } from "@webstudio-is/css-data";
 import { useMemo } from "react";
+import type {
+  Breakpoint,
+  CssRule,
+  Style,
+  StyleProperty,
+  StyleValue,
+} from "@webstudio-is/css-data";
+import { properties } from "@webstudio-is/css-data";
+import { utils } from "@webstudio-is/project";
+import type { Instance } from "@webstudio-is/react-sdk";
+import { useBreakpoints, useRootInstance } from "~/shared/nano-states";
+import {
+  useSelectedBreakpoint,
+  useSelectedInstanceData,
+} from "~/designer/shared/nano-states";
+
+type Cascaded = {
```

Comment:
Trying to find a more precise name for this, CascadedValueContainer is very correct and very ugly

Reply:
Maybe CascadedInfo

Reply:
or CascadedValueInfo

Reply:
I don't think we will add "ValueContaine" to every computed value. This is an ugly part tbh.

Reply:
something that is semantically not clear is long-term worse than something that has a long name